xclim/data/fr.json

```
@@ -463,6 +463,13 @@
     "comment": "",
     "abstract": "Cumul des degrés-jours pour les jours où la température moyenne est au-dessus d'un seuil."
   },
+  "BIOLOGICALLY_EFFECTIVE_DEGREE_DAYS":{
+    "long_name": "",
+    "description": "",
+    "title": "Degré-jour de croissance effective pour la productivité biologique",
+    "comment": "Formule originale pris de Gladstones 1992.",
+    "abstract": ""
```

```suggestion
    "long_name": "Degrés-jours biologiquement actifs",
    "description": "Accumulation des degrés-jours borné par Tmin > {thresh_tasmin} et Tmax-Tmin < {max_daily_degree_days}. Les degrés jours quotidiens sont corrigés selon Tmax-Tmin et selon un coefficient de longueur du jour dépendent de la laittude.",
    "title": "Degrés-jours biologiquement actifs",
    "comment": "Formule originale prise de Gladstones 1992.",
    "abstract": "Indice basé sur une relation non linéaire entre la température et les processus physiologiques. Il intègre une borne supérieure du cumul, un coefficient lié à la longueur du jour et une correction liée à l'écart journalier des températures.",
```

After moults recherches, I found this : https://tel.archives-ouvertes.fr/tel-01968041/document , p.17
Haha second page of google search, first french publication discussing BEDD :)